spec/support/spec_config.rb

```diff
@@ -92,6 +92,23 @@ def connect_options
     end
   end
 
+  def supports_scram_256?
+    return @supports_scram_256 if @supports_scram_256
```

Comment:
This is better spelled as `return @supports_scram_256 unless @supports_scram_256.nil?` which will short-circuit the rest of the method when the deployment does not support scram256.